usr.sbin/adduser/adduser.sh

```diff
@@ -890,6 +1020,7 @@ defaultLgroup=
 defaultgroups=
 defaultshell="${DEFAULTSHELL}"
 defaultHomePerm=
+zfsopt="-o compression=off -o atime=on"
```

Comment:
There should be *NO* default for creation options.  They should almost always be inherited from parent.